client/my-sites/plans/plan-overview/plan-status/index.jsx

```diff
@@ -31,6 +33,28 @@ const PlanStatus = React.createClass( {
 		page( `/checkout/${ this.props.selectedSite.slug }` );
 	},
 
+	renderNotice() {
+		const { plan } = this.props;
+
+		if ( isInGracePeriod( plan ) ) {
+			const daysAfterUserFacingExpiry = Math.abs( getDaysUntilUserFacingExpiry( plan ) ),
+				noticeText = daysAfterUserFacingExpiry === 0
+					? this.translate( 'Expired today' )
+					: this.translate(
+						'Expired %(daysAfterUserFacingExpiry)d day ago',
+						'Expired %(daysAfterUserFacingExpiry)d days ago', {
+							args: { daysAfterUserFacingExpiry },
+							count: daysAfterUserFacingExpiry
+						} );
```

Comment:
That might be just personal preference but I find this indented declaration difficult to read. What do you think of the following?

```javascript
const daysAfterUserFacingExpiry = Math.abs( getDaysUntilUserFacingExpiry( plan ) );
let noticeText;

if ( daysAfterUserFacingExpiry === 0 ) {
  noticeText = this.translate( 'Expired today' );
} else {
  noticeText = this.translate(
  'Expired %(daysAfterUserFacingExpiry)d day ago',
  'Expired %(daysAfterUserFacingExpiry)d days ago', {
    args: { daysAfterUserFacingExpiry },
    count: daysAfterUserFacingExpiry
  } );
}

return (
  <Notice isCompact status="is-error">
    { noticeText }
  </Notice>
);
```

Since we're not using `noticeText` more than once we could also imagine this shorter approach:

```javascript
const daysAfterUserFacingExpiry = Math.abs( getDaysUntilUserFacingExpiry( plan ) );

return (
  <Notice isCompact status="is-error">
    { daysAfterUserFacingExpiry === 0
      ? this.translate( 'Expired today' )
      : this.translate(
      'Expired %(daysAfterUserFacingExpiry)d day ago',
      'Expired %(daysAfterUserFacingExpiry)d days ago', {
        args: { daysAfterUserFacingExpiry },
        count: daysAfterUserFacingExpiry
      } ) }
  </Notice>
);
```